src/tests/Regressions/coreclr/GitHub_100536/test100536.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <!-- The issue this test is for can only occur on targets that allow inlining pinvokes in the same frame as the frame of the continuation of catch handler -->
+    <CLRTestTargetUnsupported Condition="'$(TargetArchitecture)' != 'arm' AND '$(TargetArchitecture)' != 'x86'">true</CLRTestTargetUnsupported>
+    <AlwaysUseCrossGen2>true</AlwaysUseCrossGen2>
```

Comment:
Why do we need set AlwaysUseCrossGen2 to true?

Reply:
Without crossgening, the pinvoke was not inlined into the method and used an IL stub instead.

Reply:
The comment in the check I've taken from the old EH says:
> If we aren't setting up the frame for each P/Invoke (instead setting up once per method), then ReadyToRun code is the only code using the per-P/Invoke logic.

Reply:
Regression tests are interesting code patterns that happened to hit a bug at one point. We want the regression tests to be as broadly applicable as possible over our test matrix. It is unlikely that the exact same bug is going to come back. It is more likely that a similar bug, potentially under a different runtime configuration or on a different architecture, may show up.

I would delete both the architecture filter and the forced crossgen from the test project (crossgen flavor will be covered by the outer loop test matrix).

Reply:
Done